tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -53,6 +53,7 @@ user signs out
 
 user selects theme and topic from admin dashboard
     [Arguments]  ${theme}  ${topic}
+    user goes to url  %{ADMIN_URL}
```

Comment:
I think in most instances this keyword is used, it is assumed that the user is already on the admin dashboard. So I think this is going to be redundant in most cases. I think I'd prefer that the test suite itself navigates the user to the dashboard.

Reply:
I'd prefer to keep it because in quite a few cases it stops cascade failures when one step fails which in turn stops the user navigating to the dashboard, and then causes multiple failures off the back of it that could be avoided.

How about I make it conditional so that if you're already on the dashboard, it doesn't navigate there?

Reply:
> I'd prefer to keep it because in quite a few cases it stops cascade failures

Cool - not that about that kind of thing before - makes sense. :)

That sounds good. To be honest, if you leave it I don't think its a big deal.